### PR TITLE
329: The Trophy of Awesome

### PR DESCRIPTION
**Pull request by pimotte**
In order to play on forever. The second part is just to spice things up a little bit, which I feel we can do if it doesn't result in the whole game ending forever.

勝ちましたか？ = Did you win?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +232,20 @@ This rule takes precedence over every other rule determining the winner.
 **213** A non-player can become a player by submitting a pull request which adds
 their name and Github handle to the Player List.
 
+**329** *The Trophy of Awesome*
+
+Whenever a player would win the game, instead, a virtual ``Trophy of Awesome''
```
Comment (MrHug) [inline README.md]: No need to use LaTeX ticks

**Comment by MrHug** [comment]
I am generally in favour of this ;)

**Comment by pimotte** [comment]
Fixed the quotes.

**Comment by MrHug** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by jdonkervliet** [comment]
Before I vote on this I would like to have the rule-change changed in such away that the number of points of the 'winner' is reset (in a fair way). Otherwise, resetting the game state to where a player has 199 points forces the other players to either 1) block the pull requests of the winner or 2) quickly create 'messy' rule-changes to increase the number of points in the win condition or decrease the number of points of the winner.

**Comment by pimotte** [comment]
Sounds reasonable.

**Comment by MrHug** [comment]
:+1:

**Comment by jdonkervliet** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by pimotte** [comment]
Since above commit is a merge commit, with my implicit vote, we have 3/4, so I'm merging this.

**Pull request merged**